tests/api/sru/test_sru_live.py

```
@@ -0,0 +1,501 @@
+def sru(query, maximum_records=3, start_record=1, fmt=None):
+    """Perform a searchRetrieve request and return the parsed XML dict."""
+    params = {
+        "version": "1.1",
+        "operation": "searchRetrieve",
+        "query": query,
+        "maximumRecords": maximum_records,
+        "startRecord": start_record,
+    }
+    if fmt:
+        params["format"] = fmt
+    resp = requests.get(BASE_URL, params=params, verify=False, timeout=10)
```

Ruff (0.15.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)